R/create_extendr_package.R

```diff
@@ -0,0 +1,80 @@
+
+#' Create a project for R package development with Rust
+#'
+#' @description
+#' This function creates an R project directory for package development
+#' with Rust extensions.
+#'
+#' @param path a path to new directory
+#' @param ... arguments passed on to `usethis::create_package()` and
+#' `rextendr::use_extendr()`
+#'
+#' @return Path to the newly created project or package, invisibly.
+#' @keywords internal
+#'
+#' @examples
+create_extendr_package <- function(path, ...) {
+
+  # error if usethis is not installed
+  rlang::check_installed("usethis")
+
+  args <- rlang::list2(...)
+
+  # hunch is that rstudio project text input widgets return empty strings
+  # when no value is given, want to make sure it is NULL so `use_extendr()`
+  # handles it correctly
+  nullify_empty_string <- function(x) {
+    if (rlang::is_string(x) && nzchar(x)) x else NULL
+  }
+
+  args <- purrr::map(args, nullify_empty_string)
+
+  # build package directory, but don't open yet!
```

Comment:
What does 'open' mean here?

Reply:
That's a parameter in `usethis::create_package()`. Evidently, it can mean several different things depending on the context, but it basically means "open the new project in a new R session or change the active project in RStudio to the new one." I believe we want to set it to FALSE and let the RStudio project GUI handle that instead. But, do you want me to add the text from the [{usethis} docs](https://usethis.r-lib.org/reference/create_package.html) to that comment for clarification?

Reply:
Can you just rephrase it so it is better understood by people not involved with RStudio?

Reply:
Yes! Will do!

Reply:
this should be done with recent commit, let me know if that is sufficient